*** /data/roundcubemail/password.inc.php	2015-06-18 14:12:25.705517061 +0000
--- /etc/roundcubemail/password.inc.php	2015-07-08 13:35:37.000000000 +0000
***************
*** 45,51 ****
  
      // LDAP base name (root directory)
      // Exemple: 'dc=exemple,dc=com'
!     $config['password_ldap_basedn'] = 'dc=example,dc=org';
  
      // LDAP connection method
      // There is two connection method for changing a user's LDAP password.
--- 45,51 ----
  
      // LDAP base name (root directory)
      // Exemple: 'dc=exemple,dc=com'
!     $config['password_ldap_basedn'] = 'ou=People,dc=example,dc=org';
  
      // LDAP connection method
      // There is two connection method for changing a user's LDAP password.
***************
*** 98,105 ****
      // LDAP search base
      // If password_ldap_searchDN is set, the base to search in using the filter below.
      // Note that you should comment out the default password_ldap_userDN_mask setting
      // for this to take effect.
!     $config['password_ldap_search_base'] = 'dc=example,dc=org';
  
      // LDAP search filter
      // If password_ldap_searchDN is set, the filter to use when
--- 98,105 ----
      // LDAP search base
      // If password_ldap_searchDN is set, the base to search in using the filter below.
      // Note that you should comment out the default password_ldap_userDN_mask setting
      // for this to take effect.
!     $config['password_ldap_search_base'] = 'ou=People,dc=example,dc=org';
  
      // LDAP search filter
      // If password_ldap_searchDN is set, the filter to use when
